cmd/gearlogger/main.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"net"
+	"net/url"
+	"time"
+
+	"github.com/Clever/gearadmin"
+	"gopkg.in/Clever/kayvee-go.v1"
+)
+
+func logMetrics(g gearadmin.GearmanAdmin) {
+	statuses, err := g.Status()
+	if err != nil {
+		log.Fatalf("error retrieving gearman status: %s", err)
+	}
+	for _, status := range statuses {
+		fmt.Println(kayvee.FormatLog("gearlogger", "info", "status", map[string]interface{}{
+			"function": status.Function,
+			"running":  status.Running,
+			"total":    status.Total,
+			"workers":  status.AvailableWorkers,
+		}))
```

Comment:
As far as getting this picked up by influx/grafana, you've got two options here:
1) add "type": "gauge"
2) write a custom matcher for this message type in the data.toml config

I'd probably go with 1 because it's easier, and will lead to this being picked up automatically.

Also, where will this be getting called from?  Stdout will not automatically go to syslog unless it is being run by supervisor (or in a container being managed by supervisor).  Also, we tend to prefer stderr (log.Println) but both should get forwarded.